internal/plugins/helm/v1/init.go

```diff
@@ -20,10 +20,11 @@ import (
 	"path/filepath"
 	"strings"
 
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
```

Comment:
This import is still not ordered. Right group, but wrong order (should be last).